### ISSUE TITLE
Debezium CDC json unused `schema` field

### ISSUE THREAD

**Issue by WillyKidd** [risingwavelabs/risingwave#9891]
For the mysql and postgres debezium connectors, the generated `json` (each cdc message) contains the `schema` field, which contains information about the upstream schema.However this field is not used by risingwave. Instead, the  json parser on the RW side uses the schema provided by the user to parse the cdc message.

Also, since rw uses grpc to communicate with the connector node, the `schema` field generates unused network traffic, which might have an impact on the connector throughput.

For complicated schemas, it is possible that that the `schema` field takes up a great portion of each CDC message. Maybe we should remove it.

**Comment by BugenZhao** [comment]
By the way, I'm wondering if we can use a binary format to replace JSON for the protocol between the connector node and RisingWave... 🤔

**Comment by StrikeW** [comment]
Yes, we can. Technically, Debezium can output the events in Avro format, and recently there are some works on our Debezium Avro parser. In the current stage, I think JSON format is easier to track down issues. And we can evaluate the performance of both formats when direct CDC become more mature.
